### ISSUE TITLE
[Feature Request]: Image Processing Module for CNN and Image Recognition

### ISSUE THREAD

**Issue by DarshAgrawal14** [UTSAVS26/PySnippets#148]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

When training Convolutional Neural Networks (CNNs) or image recognition models, one of the major challenges is effectively augmenting the training data to improve model performance and generalization. Data augmentation plays a crucial role in enhancing the diversity of the training dataset, which helps prevent overfitting and allows the model to learn more robust features.

Creation of  a module containing functions to handle :
- Rotation
- Flipping
- Scaling
- Cropping
- Color adjustments
- Resizing
- Changing format
- Normalization etc

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

### Full Name

Darsh Agrawal

### Participant Role

GSSOC
